### ISSUE TITLE
Power strips for workshop room

### ISSUE THREAD

**Issue by mqwilber** [mqwilber/2015-04-17-ucsb#16]
@TomCSmith, @umihoshijima, @tombellkelp, @lstevenson09 if you have extra power strips in the lab or at home, could you try to bring them for the workshop on Friday?  The room has plenty of outlets and we may not need the power strips, but I think that having a few will make it easier for people to plugin and easier for us to move around

**Comment by umihoshijima** [comment]
Yep! will bring them on friday, or if you want me to tag along to the room checkout on thurdsay.

**Comment by lstevenson09** [comment]
Yep! We have tons in the lab. I can drop them off at your office if that’s easier, Mark, or bring them on Friday

**Comment by mqwilber** [comment]
Excellent! Yeah, if people could bring them by my office on Thursday sometime that would be great. @umihoshijima help setting up the room on thursday would be much appreciated. Thanks!
